src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/SetAzureSqlDatabaseBackupLongTermRetentionPolicy.cs

```diff
@@ -70,6 +70,10 @@ public class SetAzureSqlDatabaseBackupLongTermRetentionPolicy : AzureSqlDatabase
         /// <summary>
         /// Gets or sets the name of the backup long term retention policy
         /// </summary>
+        /// <remarks>
+        /// The upcoming breaking change will need some logic changes of how the ResourceId is handled. Contact adeal or pixia for assistance.
+        /// </remarks>
+        [CmdletParameterBreakingChange("ResourceId", "ResourceId will no longer refer to RecoveryServicesBackupPolicyResourceId as it is no longer needed. Instead, it will refer to the ResourceId of the long term retention policy being set.")]
```

Comment:
As discussed, plan is to remove this parameter for now and we will re-add it to have behavior that references correct resourceId (rather than deprecated RecoveryServicesBackupPolicyResourceId)